pcsx2/GS/Renderers/HW/GSRendererNew.cpp

```diff
@@ -1574,7 +1577,6 @@ void GSRendererNew::DrawPrims(GSTexture* rt, GSTexture* ds, GSTextureCache::Sour
 			return;
 
 		// RenderHW always renders first pass, replace first pass with second
-		memcpy(&m_conf.cb_ps,     &m_conf.alpha_second_pass.cb_ps,     sizeof(m_conf.cb_ps));
```

Comment:
`m_conf.cb_ps.FogColor_AREF.a = m_conf.alpha_second_pass.ps_aref`

Though tbh I'm fairly sure we have our constants set up so opposite atst pairs will always use the same aref constant
So if you wanted you could probably get rid of `m_conf.alpha_second_pass.ps_aref` completely